src/TwigHooks/src/Debug/DebugAwareRendererInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\TwigHooks\Debug;
+
+interface DebugAwareRendererInterface
+{
+    public const CONTEXT_DEBUG_PREFIX = '_debug_prefix';
+
+    public const CONTEXT_DEBUG_SUFFIX = '_debug_suffix';
```

Comment:
```patch
---    public const CONTEXT_DEBUG_PREFIX = '_debug_prefix';
---
---    public const CONTEXT_DEBUG_SUFFIX = '_debug_suffix';
+++    public const CONTEXT_KEY_DEBUG_PREFIX = '_debug_prefix';
+++
+++    public const CONTEXT_KEY_DEBUG_SUFFIX = '_debug_suffix';
```

Reply:
I'm not familiar with context key-value format, is underscore acceptable part of the key?

Maybe like so?
```patch
--- _debug_prefix
+++ debug_prefix
```

Reply:
Another point, it's a debug comment, not just debug. How about this?

```php
    public const CONTEXT_KEY_DEBUG_COMMENT_PREFIX = '_debug_comment_prefix';

    public const CONTEXT_KEY_DEBUG_COMMENT_SUFFIX = '_debug_comment_suffix';
```

Reply:
> I'm not familiar with context key-value format, is underscore acceptable part of the key?
> 
> Maybe like so?
> 
> ```diff
> --- _debug_prefix
> +++ debug_prefix
> ```

Starting with underscore means it's a specific value which is outside of common templates variables. `_prefixes` are handled with a starting underscore for example.

Reply:
> Another point, it's a debug comment, not just debug. How about this?
> 
> ```
>     public const CONTEXT_KEY_DEBUG_COMMENT_PREFIX = '_debug_comment_prefix';
> 
>     public const CONTEXT_KEY_DEBUG_COMMENT_SUFFIX = '_debug_comment_suffix';
> ```

I agree with this, I was only worried about the length of this key since you will have to write it fully, that's why I kept it shorter.